src/Raven.Server/ServerWide/Commands/UpdateTopologyCommand.cs

```diff
@@ -1,24 +1,29 @@
-﻿using Raven.Client.ServerWide;
+﻿using System;
+using Raven.Client.ServerWide;
 using Sparrow.Json.Parsing;
 
 namespace Raven.Server.ServerWide.Commands
 {
     public class UpdateTopologyCommand : UpdateDatabaseCommand
     {
         public DatabaseTopology Topology;
+        public DateTime At;
```

Comment:
this can be DateTime.Min for old commands?

Reply:
yes, handle this case in the `ClusterObserver`